iroh-bytes/Cargo.toml

```diff
@@ -13,15 +13,15 @@ rust-version = "1.65"
 
 [dependencies]
 anyhow = { version = "1", features = ["backtrace"] }
-bao-tree = { version = "0.4.1", features = ["tokio_fsm"], default-features = false }
+bao-tree = { version = "0.5.0-rc.1", features = ["tokio_fsm"], default-features = false }
 blake3 = "1.3.3"
 bytes = { version = "1.4", features = ["serde"] }
 data-encoding = "2.3.3"
 derive_more = { version = "0.99.17", git = "https://github.com/JelteF/derive_more", features = ["debug", "display", "from", "try_into"] }
 flume = "0.10.14"
 futures = "0.3.25"
 hex = "0.4.3"
-iroh-io = { path = "../iroh-io" }
+iroh-io = { version = "0.2.0-rc.1" }
```

Comment:
this should be both, so in development mode it always refers to the local version, otherwise things get quite problematic, especially with releases

Reply:
yeah, the problem is that bao-tree, which is not in this repo, depends on iroh-io, which is in this repo. This is a bit of a mess. Either pull bao-tree in, make iroh-io a separate repo, or remove the stuff that uses iroh-io from bao-tree and put it somewhere here.

Not quite sure what to do about it.